src/TwigComponent/tests/Fixtures/templates/anonymous_component_none_scalar_prop.html.twig

```diff
@@ -0,0 +1 @@
+<twig:UserCard :user='user'/>
```

Comment:
We should pass some real attribute here also - like `class="foo"` and test that it is rendered